locksmith/src/operations/metadataOperations.ts

```diff
@@ -7,10 +7,8 @@ import networks from '@unlock-protocol/networks'
 import { Verifier } from '../models/verifier'
 import Normalizer from '../utils/normalizer'
 import * as lockOperations from './lockOperations'
-import * as Asset from '../utils/assets'
 import { Attribute } from '../types'
 import metadata from '../config/metadata'
-const baseURIFragement = 'https://assets.unlock-protocol.com'
```

Comment:
So the endpoint does not work and all the methods which use these to generate an some kind of base metadata endpoint do not work either so I removed it. Was this intended to be aws s3?

Reply:
No this is legacy stuff... ok to remove!